src/test/java/pages/ProfilePage.java

```diff
@@ -0,0 +1,53 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+
+import java.util.UUID;
+
+public class ProfilePage extends BasePage{
+    public ProfilePage (WebDriver givenDriver) {
+        super(givenDriver);
+    }
+    @FindBy(css="#inputProfileCurrentPassword")
+    WebElement currentPasswordField;
+    @FindBy(css="#inputProfileName")
+    WebElement usernameField;
+    @FindBy(css = "#inputProfileEmail")
+    WebElement emailField;
+    @FindBy(css = "#inputProfileNewPassword")
+    WebElement newPasswordField;
+    @FindBy(css = "button.btn-submit")
+    WebElement saveBtn;
+    @FindBy(css="a.view-profile>span.name")
+    WebElement actualProfileName;
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```
   @FindBy(css = "img.avatar")
    private WebElement avatar
```